src/test/java/org/fundacionjala/coding/SolutionTest.java

```diff
@@ -0,0 +1,110 @@
+package org.fundacionjala.coding;
+
+//import java.util.Random;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)